openclaw_hooks/nima-memory/ladybug_store.py

```
@@ -215,132 +219,45 @@ def _store_memory_attempt(data: dict, conn, retry_offset: int = 0) -> tuple[bool
-                print(f"error:max retries ({max_retries}) exceeded due to ID conflicts", file=sys.stderr)
-                return False
-                
-        finally:
-            try:
-                conn.close()
-            except:
-                pass
-    
-    return False
+            conn.close()
+        except:
```

Ruff (0.15.1)

[error] Do not use bare `except`

(E722)